spotseeker_server/models.py

```diff
@@ -211,6 +210,12 @@ def get_with_external(spot_id):
         else:
             return Spot.objects.get(pk=spot_id)
 
+    def delete(self):
```

Comment:
This approach should be used on edit as well, to make sure the cache is fresh.  that way you don't need to validate etags in your get methods.